redirects/16.0.txt

```diff
@@ -91,7 +91,7 @@ applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst ap
 applications/inventory_and_mrp/inventory/routes/strategies.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/advanced_operations_warehouse.rst                                             # /inventory_and_mrp/inventory/routes/strategies -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/advanced_operations_warehouse
 applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/putaway.rst                                                  # /inventory_and_mrp/inventory/routes/strategies/* -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
 applications/inventory_and_mrp/inventory/routes/strategies/removal.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst                                                  # /inventory_and_mrp/inventory/routes/strategies/* -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
-applications/inventory_and_mrp/inventory/management/products/usage.rst applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/usage.rst                                                    # /inventory_and_mrp/inventory/management/products/usage -> /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/*
+applications/inventory_and_mrp/inventory/management/products/usage.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst                                                              # /inventory_and_mrp/inventory/management/products/usage -> /inventory_and_mrp/inventory/product_management/product_replenishment/uom/*
```

Comment:
Nitpick– this is a file-to-file redirect instead of a whole folder-to-folder, so the `/*` is not needed :)
```suggestion
applications/inventory_and_mrp/inventory/management/products/usage.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst                                                              # /inventory_and_mrp/inventory/management/products/usage -> /inventory_and_mrp/inventory/product_management/product_replenishment/uom
```